projects/addon-doc/tokens/url-lower-case.ts

```diff
@@ -0,0 +1,3 @@
+import {tuiCreateToken} from '@taiga-ui/cdk';
+
+export const TUI_DOC_URL_LOWER_CASE = tuiCreateToken(false);
```

Comment:
Looks overengineered to me. How about we just provide serializer in demo without all that configurations?

Reply:
fixed

Reply:
I mean let's do it in demo app, because people might not want this behavior in addon-doc.

Reply:
But I wanna use it in tui-editor demo. Will I need to duplicate it?